src/Simplex/Chat/Migrations/M20231219_notes_folders.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20231219_notes_folders where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20231219_notes_folders :: Query
+m20231219_notes_folders =
+  [sql|
+    CREATE TABLE notes_folders (
+      notes_folder_id INTEGER PRIMARY KEY AUTOINCREMENT,
+      user_id INTEGER NOT NULL REFERENCES users ON DELETE CASCADE,
+      local_display_name TEXT NOT NULL,
+      created_at TEXT NOT NULL,
+      updated_at TEXT NOT NULL,
+      chat_ts TEXT,
+      favorite INTEGER NOT NULL DEFAULT 0,
+      unread_chat INTEGER DEFAULT 0 NOT NULL,
+      chat_item_id INTEGER DEFAULT NULL REFERENCES chat_items ON DELETE SET NULL,
```

Comment:
this field is unnecessary - if I remember correctly it's only used in groups to refer to invitation item
```suggestion
```